ssl/quic/quic_port.c

```diff
@@ -1022,11 +1177,15 @@ static int port_validate_token(QUIC_PKT_HDR *hdr, QUIC_PORT *port,
     int ret = 0;
     QUIC_VALIDATION_TOKEN token = { 0 };
     unsigned long long time_diff;
-    size_t remote_addr_len;
-    unsigned char *remote_addr = NULL;
+    size_t remote_addr_len, dec_token_len;
+    unsigned char *remote_addr = NULL, dec_token[MARSHALLED_TOKEN_MAX_LEN];
     OSSL_TIME now = ossl_time_now();
 
-    if (!parse_validation_token(&token, hdr->token, hdr->token_len))
+    if (!decrypt_validation_token(port, hdr->token, hdr->token_len, NULL,
+                                  &dec_token_len)
```

Comment:
Please check dec_token_len here.

Reply:
It's already checked within `decrypt_validation_token`: https://github.com/openssl/openssl/pull/26165/files#diff-041458c600186ca5d159cfe8a41037b2590e4aac948d12d3520e813d2524317aR865

Reply:
You check the encrypted token len there. You should check the decrypted len here - I am not saying that such added check can really fail but still for code sanity it is better to add such check.